src/features/classification/security-controls/utils.js

```diff
@@ -10,58 +10,63 @@ import downloadRestrictionsMessageMap from './downloadRestrictionsMessageMap';
 import messages from './messages';
 import { ACCESS_POLICY_RESTRICTION, DOWNLOAD_CONTROL, LIST_ACCESS_LEVEL, SHARED_LINK_ACCESS_LEVEL } from '../constants';
 
+type MessageItem = {
+    message: ?MessageDescriptor,
+    tooltipMessage?: MessageDescriptor,
+};
+
 const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP } = ACCESS_POLICY_RESTRICTION;
 const { DESKTOP, MOBILE, WEB } = DOWNLOAD_CONTROL;
 const { BLOCK, WHITELIST, BLACKLIST } = LIST_ACCESS_LEVEL;
 const { COLLAB_ONLY, COLLAB_AND_COMPANY_ONLY, PUBLIC } = SHARED_LINK_ACCESS_LEVEL;
 
-const getShortSecurityControlsMessage = (controls: Controls): ?MessageDescriptor => {
+const getShortSecurityControlsMessage = (controls: Controls): MessageItem => {
```

Comment:
This could return a type `?MessageItem`. The structure of the object returned has changed, but I don't think it's necessary to change the logic of returning null when there's no message.